meta/classes/base.bbclass

```diff
@@ -84,7 +84,7 @@ base_update_conf_value() {
         echo "${key}${key_prefix}=" >> "${config_file}"
     fi
     # substitute
-    sed -i "s%\(^${key} *[?:]*= *\).*%\1\"${value}\"%" "${config_file}"
+    sed -i "s%\(^${key} *[?+:]*= *\).*%\1\"${value}\"%" "${config_file}"
```

Comment:
Why you didn't squashed this change with https://github.com/xen-troops/xt-distro/commit/a4d4fd577f2101a1c37360b02c6576ce65f6f66a?